keploy.sh

```diff
@@ -35,7 +35,7 @@ installKeploy (){
             download_url="https://github.com/keploy/keploy/releases/latest/download/keploy_darwin_all.tar.gz"
         fi
 
-        curl --silent --location "$download_url" | tar xz -C /tmp
+        sudo curl --silent --location "$download_url" | sudo tar xz -C /tmp
```

Comment:
@suvanbanerjee sudo isn't required please refer to the discussion we had on issue #1949

Reply:
Sorry for not reading the comments well. I have two proposed solutions:

1. Before extraction, remove the problematic files from the `/tmp` directory.
2. Use the `--overwrite` flag with the tar command.

I think using the `--overwrite` flag is better. What is your opinion?